### PR TITLE
Add example for Interaction Energy calculation

### PR DESCRIPTION
**Pull request by edmund-qcware**

### CODE REVIEW

File: examples/interaction_energy/sdk/run.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+from promethium_sdk.utils import base64encode
+
+foldername = "output"
+gpu_type = os.getenv("PM_GPU_TYPE", "a100")
+
+if not os.path.exists(foldername):
+    os.makedirs(foldername)
+
+moleculeA_adenine = base64encode(
+"""
```
Comment (pdbaines) [inline examples/interaction_energy/sdk/run.py:16]: Let's make these properly formatted XYZ files i.e., 2-line header with atom count. While this "lazy" format is accepted, it's preferable to use the proper format

File: examples/interaction_energy/sdk/run.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+workflow = prom.workflows.get(workflow.id)
+print(f"Workflow {workflow.name} completed with status: {workflow.status}")
+print(f"Workflow completed in {workflow.duration_seconds:.2f}s")
+
+ie_results = prom.workflows.results(workflow.id)
+with open(f"{foldername}/{workflow.name}_results.json", "w") as fp:
+    fp.write(ie_results.model_dump_json(indent=2))
+
+# Numeric results:
+interaction_energy = 627.5095 * ie_results.results["interaction_energy"]["raw_interaction_energy"]
```
Comment (pdbaines) [inline examples/interaction_energy/sdk/run.py:134]: Be nice to pull this into a constant and have the variable name explain what conversion is being done

**Review by pdbaines — APPROVED** [review]
LGTM, couple of minor suggestions

File: examples/interaction_energy/sdk/run.py (hunk 1 above)
Comment (edmund-qcware) [reply examples/interaction_energy/sdk/run.py]: Done!

File: examples/interaction_energy/sdk/run.py (hunk 2 above)
Comment (edmund-qcware) [reply examples/interaction_energy/sdk/run.py]: Done!

**Pull request merged**